.github/workflows/japi-compliance-checker.yml

```diff
@@ -0,0 +1,85 @@
+name: Check API compatibility between two jars
+
+on:
+  workflow_call:
+    inputs:
+      runsOn:
+        required: false
+        type: string
+        default: 'ubuntu-20.04'
+      javaVersion:
+        required: false
+        type: string
+        default: '11'
+      javaVendor:
+        required: false
+        type: string
+        default: 'zulu'
+      targetJarUrl:
+        required: true
+        type: string
+        default: ""
+        description: 'url to the target api version jar file.'
+      projectPath:
+        required: false
+        type: string
+        default: '.'
+        description: "path to the Gradle module. Root directory by default"
+      gradlePropertiesPath:
+        required: false
+        type: string
+        default: '.'
+        description: "path to the Gradle module. Root directory by default"
+      libsFolderPath:
+        required: false
+        type: string
+        default: '.'
+        description: "custom path to the library. Same as projectPath/build/libs by default"
+
+
+jobs:
+  japi-compliance-check:
+    runs-on: ${{ inputs.runsOn }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get release_version
+        id: ver
+        uses: christian-draeger/read-properties@1.1.1
+        with:
+          path: ${{ inputs.gradlePropertiesPath }}/gradle.properties
+          properties: release_version
+
+      - name: Set up JDK ${{ inputs.javaVendor }} ${{ inputs.javaVersion }}
+        uses: actions/setup-java@v3
+        with:
+          distribution: ${{ inputs.javaVendor }}
+          java-version: ${{ inputs.javaVersion }}
+
+      - name: install japi-compliance-checker
+        run: sudo apt-get update && sudo apt-get install -y japi-compliance-checker
+
+      - name: Download target jar
+        run: curl -o targetVersion.jar -L ${{ inputs.targetJarUrl }}
+
```

Comment:
Please add gradle setup before executing it
```
      - name: Setup Gradle
        uses: gradle/gradle-build-action@v2
```
Maybe there is a later version of this action. Could you please check?

Reply:
Seems like v2 is the latest